pyinfra/api/facts.py

```diff
@@ -69,6 +69,9 @@ def __init__(cls, name, bases, attrs):
 class FactBase(object):
     abstract = True
 
+    def shell_executable(self):
```

Comment:
This should be an attribute of the class, not method, so just:

```
class FactBase(object):
    ...
    shell_executable = None
```

Reply:
Ok